core/src/main/resources/lib/platform/stm32/arm-none-eabi-gcc.cmake

```diff
@@ -0,0 +1,23 @@
+set(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR arm)
+
+# Some default GCC settings
+set(TOOLCHAIN_PREFIX arm-none-eabi-)
+set(FLAGS
```

Comment:
Does this really belong here? E.g. `--specs=nano.specs` instructs it to link with newlib nano. Maybe the user wants full newlib, or another library altogether?

Reply:
this im not super sure about, I essentially copied this over from the generic STM32 toolchain so I assumed it should be correct

Reply:
OK, then it is probably fine to keep it in the toolchain fiile